src/resources/js/v2/tickets-commerce.js

```diff
@@ -84,6 +88,37 @@ tribe.tickets.commerce = {};
 		$document.on( obj.customEvents.hideLoader, obj.loaderHide );
 	}
 
+	/**
+	 * Display the notice component.
+	 *
+	 * @since TBD
+	 *
+	 * @param {Object} data Data for notice.
+	 */
+	obj.showNotice = function( data = {} ) {
+		const $notice = $( obj.selectors.noticeItem );
+		const $content = $notice.find( obj.selectors.noticeContent );
+		const $title = $notice.find( obj.selectors.noticeTitle );
+
+		const title = data.title ?? tecTicketsCommerceData.default_notice.title;
```

Comment:
[linter is giving an error here.](https://github.com/the-events-calendar/event-tickets/pull/2095/checks?check_run_id=3719610883#step:6:22)